adserver/templates/adserver/publisher/overview.html

```diff
@@ -0,0 +1,78 @@
+{% extends "adserver/base.html" %}
+{% load crispy_forms_tags %}
+{% load humanize %}
+{% load i18n %}
+
+
+{% block title %}{% trans 'Publisher overview' %} - {{ publisher }}{% endblock %}
+
+
+{% block breadcrumbs %}
+  {{ block.super }}
+
+  {% url 'publisher_main' publisher.slug as publisher_main_url %}
+  {% if request.path == publisher_main_url %}
+    <li class="breadcrumb-item active">{{ publisher }}</li>
+  {% else %}
+    <li class="breadcrumb-item"><a href="{{ publisher_main_url }}">{{ publisher }}</a></li>
+  {% endif %}
+
+{% endblock breadcrumbs %}
+
+
+{% block content_container %}
+  <section>
+    <h1>{% blocktrans %}{{ publisher }}{% endblocktrans %}</h1>
+    <p><small>Month to date overview for {{ publisher }}.</small></p>
```

Comment:
I feel like this is a bit hidden. I'm looking at data for April 1, and like "oh no, my revenue is so low!" -- not sure if we have a better solution there though.

Reply:
I think I'll put the month in the H1. It isn't perfect but it's better than nothing.